bot-get.js

```diff
@@ -31,4 +31,124 @@ There will be multiple "regions" and "locations", but the Progression region wil
 We can guarantee a 100% chance.
 
 
-*/
\ No newline at end of file
+*/
+
+var goal = false;
+
+var server = require('./server.js');
+var webhook = require('./webhook-put.js');
+var chatipelago = require('./archipelagoHelper.js');
```

Comment:
this variable named come from how 1 had 1st name the helper file, before renaming it, maybe rename the variable too